runtime/src/accounts_index.rs

```diff
@@ -1185,6 +1185,7 @@ impl<T: 'static + Clone + IsCached + ZeroLamport> AccountsIndex<T> {
         max_clean_root: Option<Slot>,
         account_indexes: &HashSet<AccountIndex>,
     ) {
+        let mut empty_key = false;
```

Comment:
nit: prefix booleans with a predicate like 'is`, i.e. rename `empty_key` -> `is_empty_key` or `is_slot_list_empty`

Reply:
Done!